src/components/Slider/_Slider.scss

```diff
@@ -0,0 +1,183 @@
+  box-shadow: 1px 1px 4px .2px rgba(0, 0, 0, 0.5);
+}
+
+.slider-md {
+  width: 200px;
+}
+
+
+.slider-lg {
+  width: 400px;
+}
+
+.slider-lg .slider-value__num-box{
+  height: 25px;
+  width: 25px;
+  font-size: 13px;
+}
+
+.slider-lg input {
+  height:12px;
+}
+
+.slider-lg input::-webkit-slider-thumb {
+  width: 20px;
+  height: 20px;
+  box-shadow: 1px 1px 4px .2px rgba(0, 0, 0, 0.5);
+}
+
+.slider-value-cont {
+  margin-bottom: 15px;
+  width: 95%;
+}
+
+.slider-value__num-box {
+  background-color: $gray-500;
+  transform: rotate(45deg);
+  position: relative;
+  width: 18px;
+  height: 18px;
+  border-radius: 50% 50% 0 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  left: var(--bubble);
+  font-size: 10px;
+}
+
+.slider-value__num{
+  transform: rotate(-45deg);
+}
+
+.slider {
+  -webkit-appearance: none;
+  appearance: none;
+  background: transparent;
+  cursor: pointer;
+  margin-left: 5px;
+  width: 100%;
+  outline: 0;
+  height: 7px;
+  border-radius: 40px;
+  background: linear-gradient(to right, $gray-500 0%, $gray-500 var(--progress), $gray-100 var(--progress), $gray-100 100%);
+  z-index: 0;
+}
+
+.slider-disabled {
+  pointer-events: none;
+}
+
+.slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  width: 12px;
+  height: 12px;
+  background: $gray-500;
+  border-radius: 50%;
+  box-shadow: 1px 1px 4px .2px rgba(0, 0, 0, 0.5);
+}
+
+.slider::-webkit-slider-thumb:hover {
+  box-shadow: 0px 0px 0px 6px rgba($color: $gray-500, $alpha: .4);
+}
+
+.slider::-webkit-slider-thumb:active {
+  box-shadow: 0px 0px 0px 10px rgba($color: $gray-500, $alpha: .4);
+}
+
+
+//PRIMARY
+.slidernum-primary{
+  background-color: $primary;
+  color: white;
+}
+
+.slider-primary {
+  background: linear-gradient(to right, $primary 0%, $primary var(--progress), $light-primary var(--progress), $light-primary 100%);
+}
+
+.slider-primary::-webkit-slider-thumb {
+  background: $primary;
+}
+
+.slider-primary::-webkit-slider-thumb:hover {
+  box-shadow: 0px 0px 0px 6px rgba($color: $primary, $alpha: .4);
+}
+
+.slider-primary::-webkit-slider-thumb:active {
+  box-shadow: 0px 0px 0px 10px rgba($color: $primary, $alpha: .4);
+}
+
+//SECONDARY
+.slidernum-secondary{
+  background-color: $secondary;
+  color: white;
+}
+
+.slider-secondary {
+  background: linear-gradient(to right, $secondary 0%, $secondary var(--progress), $light-secondary var(--progress), $light-secondary 100%);
+}
+
+.slider-secondary::-webkit-slider-thumb {
+  background: $secondary;
+}
+
+.slider-secondary::-webkit-slider-thumb:hover {
+  box-shadow: 0px 0px 0px 6px rgba($color: $secondary, $alpha: .4);
+}
+
+.slider-secondary::-webkit-slider-thumb:active {
+  box-shadow: 0px 0px 0px 10px rgba($color: $secondary, $alpha: .4);
+}
+
+.tickmarks {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+}
+
+.range__ticks {
+  width: 5px;
+}
+
+.slider__range-right {
+  margin-top: -8px;
+}
+
```

Comment:
Would be better practice to use variables from _variable.scss rather than hardcoded px values to improve maintainability